openid/example/main.go

```diff
@@ -72,14 +77,16 @@ var errorTemplate = template.Must(template.New("failure").Parse(`<html>
 </html>
 `))
 
-var loginTemplate = template.Must(template.New("success").Parse(`<html>
+var loginTemplate = template.Must(template.New("success").Parse(`<!doctype html>
+<html>
 <head><title>Login Success</title></head>
 <body>
 <table>
 <tr><th>Claimed ID</th><td>{{.ID}}</td></tr>{{if .Teams}}
 <tr><th>Teams</th><td>{{.Teams}}</td></tr>
-{{end}}{{range $k, $v := .SimpleRegistration}}
+{{end}}{{range $k, $v := .SReg}}
```

Comment:
Why has this changed? Perhaps it might be worth documenting/referring to the format of the input data somewhere.

Reply:
because it previously didn't work and now it does.